pom.xml

```diff
@@ -465,6 +465,19 @@
 
       <dependency>
         <groupId>org.apache.beam</groupId>
+        <artifactId>beam-sdks-java-harness-core</artifactId>
```

Comment:
Maybe we should go with `beam-sdks-java-fn-execution` since it mirrors `beam-model-fn-execution` and would store things which both the SDK harness and Runner harness would share.

Reply:
I like it. Will do.